inst/odin/carehomes.R

```diff
@@ -16,25 +16,25 @@ initial(time) <- 0
 update(time) <- (step + 1) * dt
 ## output number of individuals vaccinated by age and clinical stage
 ## vaccinated S
-initial(n_S_vaccinated[]) <- 0
-update(n_S_vaccinated[]) <- n_S_next_vacc_class[i, 1] +
-  n_SE_next_vacc_class[i, 1]
-dim(n_S_vaccinated) <- n_groups
+initial(cumul_n_S_vaccinated[]) <- 0
```

Comment:
We've been using `cum_` as a prefix elsewhere for cumulative variables (e.g., `cum_admit_conf`) could you find-and-replace that here?

Reply:
done